verilog/tools/kythe/kythe_facts.h

```diff
@@ -101,6 +104,50 @@ struct VName {
 
 std::ostream& operator<<(std::ostream&, const VName&);
 
+// Facts for kythe.
```

Comment:
Please expand the comment.

Reply:
In particular, document where this structure/definition comes from in relation to Kythe (links to source would be good too).